src/elements/content-explorer/Content.tsx

```diff
@@ -36,6 +36,7 @@ export interface ContentProps extends Required<ItemEventHandlers>, Required<Item
     isSmall: boolean;
     isTouch: boolean;
     itemActions?: ItemAction[];
+    metadataProps?: Omit<MetadataViewContainerProps, 'currentCollection'>;
```

Comment:
looks like there should be other props omitted from here. we should use an external facing Prop type for consumers so it we don't need to keep omitting